src/executePrettier/buildEditorConfigOptions.js

```diff
@@ -1,64 +1,11 @@
 // @flow
 const _ = require('lodash/fp');
 const editorconfig = require('editorconfig');
+const editorconfigToPretter = require('editorconfig-to-prettier');
 
-type RawOpts = {
-  // $FlowFixMe
-  indent_style?: 'tab' | 'space',
-  // $FlowFixMe
-  indent_size?: 'tab' | number,
-  // $FlowFixMe
-  tab_width?: number,
-  // $FlowFixMe
-  max_line_length?: number | 'off',
-};
-
-type TranslatedOpts = {
-  tabWidth?: number,
-  printWidth?: number,
-  useTabs?: boolean,
-};
-
-const maxLineLengthIsNumber: (opts: RawOpts) => boolean = _.flow(_.get('max_line_length'), _.isNumber);
-
-const indentStyleIsTab: (opts: RawOpts) => boolean = _.conforms({ indent_style: _.isEqual('tab') });
-
-const indentSizeIsTab: (opts: RawOpts) => boolean = _.conforms({ indent_size: _.isEqual('tab') });
-
-const tabWidthIsNumber: (opts: RawOpts) => boolean = _.conforms({ tab_width: _.isNumber });
-
-const shouldUseTabWidth: (opts: RawOpts) => boolean = _.overEvery([
-  tabWidthIsNumber,
-  _.overSome([indentSizeIsTab, indentStyleIsTab]),
-]);
-
-const indentSizeIsNumber: (opts: RawOpts) => boolean = _.conforms({ indent_size: _.isNumber });
-
-const indentStyleIsSpace: (opts: RawOpts) => boolean = _.conforms({ indent_style: _.isEqual('space') });
-
-const getPrintWidth: (opts: RawOpts) => ?{ printWidth: number } = _.cond([
-  [maxLineLengthIsNumber, opts => ({ printWidth: opts.max_line_length })],
-]);
-
-const getTabWidth: (opts: RawOpts) => ?{ tabWidth: number } = _.cond([
-  [shouldUseTabWidth, opts => ({ tabWidth: opts.tab_width })],
-  [indentSizeIsNumber, opts => ({ tabWidth: opts.indent_size })],
-]);
-
-const getUseTabs: (opts: RawOpts) => ?{ useTabs: boolean } = _.cond([
-  [indentStyleIsTab, _.constant({ useTabs: true })],
-  [indentStyleIsSpace, _.constant({ useTabs: false })],
-]);
-
-const mapEditorConfigOptions = (opts?: RawOpts = {}) => ({
-  ...getPrintWidth(opts),
-  ...getTabWidth(opts),
-  ...getUseTabs(opts),
-});
-
-const buildEditorConfigOptions: (file: FilePath) => TranslatedOpts = _.flow(
+const buildEditorConfigOptions: (file: FilePath) => Object = _.flow(
```

Comment:
Now that we have so little code in this file, I feel like it's a bit silly. Can we just inline in `buildPrettierOptions`?

Then we could add a test for `buildPrettierOptions` that mocks `editorconfig` and mocks `editorconfig-to-prettier` and asserts that
- editorconfig is called with the correct arguments
- editorconfig-to-prettier is called with the correct arguments
- the results of editorconfig-to-prettier are properly merged in to the final result of `buildPrettierOptions`

Reply:
i kept it in because buildPrettierEslintOptions had a similarly silly number of lines haha.  I'll make those changes though - thanks